### PR TITLE
FEATURE: follow and unfollow users

### PR DESCRIPTION
**Pull request by EJamesssss**
Hi guys pareview, same approach as likes yung ginawa ko dito, same din na yung uniqueness is on database level yung validation. Thanks!

### CODE REVIEW

**Review by reyolan — CHANGES_REQUESTED** [review]
Instead of `!@follow.save`, medyo napapangitan lang ako since laging ninenegate.

You can do `unless @follow.save`.

Basically, `unless` is opposite of `if`.

So, just for clearer comparison:

`if @follow.save` would be equivalent to: `unless !@follow.save`
`if !@follow.save` would be equivalent to: `unless @follow.save`

**Comment by EJamesssss** [comment]
> Instead of `!@follow.save`, medyo napapangitan lang ako since laging ninenegate.
> 
> You can do `unless @follow.save`.
> 
> Basically, `unless` is opposite of `if`.
> 
> So, just for clearer comparison:
> 
> `if @follow.save` would be equivalent to: `unless !@follow.save` `if !@follow.save` would be equivalent to: `unless @follow.save`

@reyolan already removed the negation on @follow.save

File: app/controllers/follows_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class FollowsController < ApplicationController
+
+    def create
+        @follow = current_user.relationships.new(follow_params)
+        if @follow.save
```
Comment (reyolan) [inline app/controllers/follows_controller.rb:5]: image
 Try this if it works, naduplicate na kasi yung redirection

Comment (EJamesssss) [reply app/controllers/follows_controller.rb]: nagwork yung unless, already pushed na rin yung change sa code 😸

**Review by reyolan — APPROVED** [review]
Looks good, nice nice!

**Pull request merged**